src/meta/api/endpoint/ApiEndPoint.ts

```diff
@@ -62,7 +62,15 @@ export const ApiEndPoint = {
   },
 
   File: {
-    biomassStock: () => apiPath('file', 'biomass-stock'),
+    biomassStock: ({
+      assessmentName = ':assessmentName',
+      cycleName = ':cycleName',
+      countryIso = ':countryIso',
+      sectionName = ':sectionName',
+      language = ':language',
+      selectedDomain = ':selectedDomain',
+    }) =>
+      apiPath('file', 'biomass-stock', assessmentName, cycleName, countryIso, sectionName, selectedDomain, language),
```

Comment:
so the url can be of format
`/file/biomass-stock/fra/2025/FIN/carbonStock/tropical/en`
instead of 
`/file/biomass-stock?assessmentName=fra&cycleName=2025&countryIso=FIN...............................`

Reply:
This is a 'link' the user can see (comparable to platform links)

API calls are invisibile for users